### PR TITLE
[BuildBot] Uplift OpenCL runtime versions

### PR DESCRIPTION
**Pull request by yinyangsx**
Signed-off-by: yinyangsx <yin.yang@intel.com>

### CODE REVIEW

**Comment by vladimirlaz** [comment]
Please make sure that RT is available and restart testing.Also please start full testing.

**Comment by vladimirlaz** [comment]
> ocl_gpu_rt_ver_win should be 27.20.100.8336
> gpu_driver_win should be 27.20.100.8336
> fpga_driver_lin should be the same as cpu_driver_lin because we are use OCLFPGA joint build in Beta08 release. The same as fpga_driver_win
> please also add OCLFPGA joint build version such as: oclfpga_ver=2020.10.7.0.15 and oclfpga_ver_win=2020.10.7.0.15
> 
> OCLCPU runtime has been uploaded on SC shared folder, and waiting for Rdrive sync.
> For OCLFPGA joint build, please ask Yanfeng to upload it on Rdrive firstly before restarting test.

@yinyangsx please make sure that proper GPU Rt is installed on windows machines and proper links are provided in comment for every RT version.

**Comment by yinyangsx** [comment]
> 
> 
> > ocl_gpu_rt_ver_win should be 27.20.100.8336
> > gpu_driver_win should be 27.20.100.8336
> > fpga_driver_lin should be the same as cpu_driver_lin because we are use OCLFPGA joint build in Beta08 release. The same as fpga_driver_win
> > please also add OCLFPGA joint build version such as: oclfpga_ver=2020.10.7.0.15 and oclfpga_ver_win=2020.10.7.0.15
> > OCLCPU runtime has been uploaded on SC shared folder, and waiting for Rdrive sync.
> > For OCLFPGA joint build, please ask Yanfeng to upload it on Rdrive firstly before restarting test.
> 
> @yinyangsx please make sure that proper GPU Rt is installed on windows machines and proper links are provided in comment for every RT version.

Windows machines have installed proper GPU RT

**Comment by yanfeng3721** [comment]
I have update the fpga emu to rdrive, please change fpga_ver=2020.10.7.0.15
fpga_ver_win=2020.10.7.0.15.

**Comment by yinyangsx** [comment]
> I have update the fpga emu to rdrive, please change fpga_ver=2020.10.7.0.15
> fpga_ver_win=2020.10.7.0.15.

The new FPGA RT is very different with the old in directory structure and location. Could you please provide the latest configuration solution?

**Comment by vladimirlaz** [comment]
> > I have update the fpga emu to rdrive, please change fpga_ver=2020.10.7.0.15
> > fpga_ver_win=2020.10.7.0.15.
> 
> The new FPGA RT is very different with the old in directory structure and location. Could you please provide the latest configuration solution?

please add both variant f directories to the path

File: buildbot/dependency.conf [hunk 1]
```diff
@@ -1,31 +1,33 @@
...
-fpga_driver_win=2020.9.2.0
+cpu_driver_lin=2020.10.7.0.15
+cpu_driver_win=2020.10.7.0.15
+gpu_driver_lin=20.25.17111
+gpu_driver_win=27.20.100.8336
+fpga_driver_lin=2020.10.7.0.15
+fpga_driver_win=2020.10.7.0.15
 # NVidia CUDA driver
 # http://developer.download.nvidia.com/compute/cuda/repos/ubuntu1804/x86_64/cuda-drivers_450.36.06-1_amd64.deb
 nvidia_gpu_driver_lin=450.36.06
```
Comment (tfzhu) [inline buildbot/dependency.conf:33]: This should be updated either as there was a downgrade

File: buildbot/dependency.conf [hunk 2]
```diff
@@ -1,31 +1,33 @@
...
-ocl_cpu_rt_ver_win=2020.10.6.0.04
+# https://github.com/intel/llvm/releases/download/2020-WW31/oclcpuexp-2020.10.7.0.15_rel.tar.gz
+ocl_cpu_rt_ver=2020.10.7.0.15
+# https://github.com/intel/llvm/releases/download/2020-WW31/win-oclcpuexp-2020.10.7.0.15_rel.zip
+ocl_cpu_rt_ver_win=2020.10.7.0.15
 # Same GPU driver supports Level Zero and OpenCL:
-# https://github.com/intel/compute-runtime/releases/tag/20.21.16886
-ocl_gpu_rt_ver=20.21.16886
+# https://github.com/intel/compute-runtime/releases/tag/20.25.17111
+ocl_gpu_rt_ver=20.25.17111
```
Comment (yanfeng3721) [inline buildbot/dependency.conf:8]: Please use the following gpu rt.
#https://github.com/intel/compute-runtime/releases/tag/20.29.17408
ocl_gpu_rt_ver=20.29.17408

**Comment by bader** [comment]
@yinyangsx, please, fix CI regressions.
It looks like new driver fixes "regression/static-buffer-dtor.cpp" - please, remove XFAIL from the test.
WRT to Windows failure it seems like environment issue @tfzhu, could you take a look, please?

**Comment by vladimirlaz** [comment]
shuffle* and common_ocl test will pass once #2219 is submitted

File: buildbot/dependency.conf [hunk 3]
```diff
@@ -1,31 +1,33 @@
...
-# https://github.com/intel/llvm/releases/download/2020-WW20/win-oclcpuexp-2020.10.6.0.4_rel.zip
-ocl_cpu_rt_ver_win=2020.10.6.0.04
+# https://github.com/intel/llvm/releases/download/2020-WW31/oclcpuexp-2020.10.7.0.15_rel.tar.gz
+ocl_cpu_rt_ver=2020.10.7.0.15
+# https://github.com/intel/llvm/releases/download/2020-WW31/win-oclcpuexp-2020.10.7.0.15_rel.zip
+ocl_cpu_rt_ver_win=2020.10.7.0.15
 # Same GPU driver supports Level Zero and OpenCL:
-# https://github.com/intel/compute-runtime/releases/tag/20.21.16886
-ocl_gpu_rt_ver=20.21.16886
+# https://github.com/intel/compute-runtime/releases/tag/20.25.17111
```
Comment (bso-intel) [inline buildbot/dependency.conf:7]: Any reason why this tag URL is different from 20.29.17408?

Comment (vladimirlaz) [reply buildbot/dependency.conf:7]: it is bug. Should be fixed to 20.29.17408

**Pull request merged**